applications/tari_console_wallet/src/main.rs

```diff
@@ -104,36 +102,36 @@ fn main_inner() -> Result<(), ExitCodes> {
         .build()
         .unwrap();
 
-    let wallet = runtime.block_on(setup_wallet(&node_config, wallet_identity))?;
+    let wallet = runtime.block_on(setup_wallet(&config, wallet_identity))?;
     debug!(target: LOG_TARGET, "Starting app");
 
     let node_identity = wallet.comms.node_identity().as_ref().clone();
     let wallet = Arc::new(RwLock::new(wallet));
     let grpc = crate::grpc::WalletGrpcServer::new(wallet.clone());
```

Comment:
We can just copy the wallet object directly, we do not need the arc or the rwlock I believe.

Reply:
Cool, I was just keeping what was already there. I'll try that out